code/modules/assembly/igniter.dm

```diff
@@ -24,6 +24,9 @@
 	var/turf/location = get_turf(loc)
 	if(location)
 		location.hotspot_expose(1000,1000)
+		for(var/T in location.contents)
+			var/atom/A = T
+			A.fire_act()

```

Comment:
Sure it does, you have an igniter in your pocket and it goes off, lighting you and your clothes on fire
what's not to get?